src/qibo/ui/mpldrawer.py

```diff
@@ -0,0 +1,687 @@
+# Some functions in MPLDrawer are from code provided by Rick Muller
+# Simplified Plotting Routines for Quantum Circuits
+# https://github.com/rpmuller/PlotQCircuit
+#
+import json
+from os import path
+from pathlib import Path
+
+import matplotlib
+import numpy as np
+
+from qibo import gates
+
+from .FusedGateBarrier import FusedEndGateBarrier, FusedStartGateBarrier
+
+UI = Path(__file__).parent
+STYLE = json.loads((UI / "styles.json").read_text())
+SYMBOLS = json.loads((UI / "symbols.json").read_text())
+
+plot_params = {
```

Comment:
```suggestion
PLOT_PARAMS = {
```
Naming convention for constants.